Add FAQ for why a library may not include cell type annotations (#306)
AlexsLemonade/scpca-docs: docs/faq.md

You can find the [function for generating a QC report](https://github.com/AlexsLemonade/scpcaTools/blob/main/R/generate_qc_report.R) and the [QC report template documents](https://github.com/AlexsLemonade/scpcaTools/tree/main/inst/rmd) in the package we developed for working with processed ScPCA data, [`scpcaTools`](https://github.com/AlexsLemonade/scpcaTools).

## Which libraries include cell type annotations?

Most single-cell and single-nuclei RNA-seq libraries available on the portal will have cell type annotations included in the processed `SingleCellExperiment` or `AnnData` object.
For more information on where to find the cell type annotations, refer to section(s) describing {ref}`SingleCellExperiment file contents <sce_file_contents:singlecellexperiment sample metadata>` and/or {ref}`AnnData file contents <sce_file_contents:anndata cell metrics>`.
If cell type annotation was performed, a supplemental cell type report (`SCPCL000000_celltype-report.html`) will be included in the download.

> There are a few scenarios where cell type annotation was not performed, which means no cell type annotations will be found in the processed objects and the download will not include a cell type report.

Reviewer: ```suggestion
There are a few scenarios where cell type annotation was not performed, which means processed objects will not include cell type annotations, and the download will not include a cell type report.
```

- Cell type annotation is not performed for cell line samples.

> - Cell type annotation is performed on the processed object which contains any cells remaining after removal of low-quality cells.
> The processed objects must contain at least 30 cells to assign cell types with `CellAssign` and at least 2 cells to assign cell types with `SingleR`.
> This means that in some rare cases there will not be enough cells remaining in the processed object for cell type annotation to be performed.

Reviewer: I would rearrange this bullet to first state where cell types are not provided and then explain why, rather than starting with the explanation.

```suggestion
- Cell type annotation is not performed on libraries with an insufficient number of cells.
The processed objects on which cell type annotation is performed must contain at least 30 cells to assign cell types with `CellAssign` and at least 2 cells to assign cell types with `SingleR`.
This means that in some rare cases there will not be enough cells remaining in the processed object for cell type annotation to be performed.
```
